src/pyubx2/ubxreader.py

```diff
@@ -109,23 +109,22 @@ def __next__(self) -> tuple:
             raise StopIteration
         return (raw_data, parsed_data)
 
-    def read(self) -> tuple:
+    def read(self, parsing=True) -> tuple:

```

Comment:
This works (more or less) if you're invoking the read() method directly, but it doesn't work if you're using the iterator. It might be better to make this a UBXReader constructor keyword argument and carry it through from there.